torch/_dynamo/symbolic_convert.py

```diff
@@ -651,7 +697,12 @@ def step(self):
             self.lineno = inst.starts_line
             self.log_starts_line()
 
-        if len(self.stack) == 0 and self.should_compile_partial_graph():
+        if (
+            len(self.stack) == 0
+            and self.should_compile_partial_graph()
+            # Inliner should not try to restore from checkpoint
```

Comment:
Hmm, is this strictly necessary?

Reply:
Sorry, which part?

Reply:
Whether inliner should not try to restore from checkpoint and compile the graph in this code path. 

It hotfixed some broken tests but I don't have a good reason to disable otherwise.